<center><img src="logo_ilum_2023.png" style="width:1000px;height:200px"/></center>

# 🥼 Caderno de experimentos de Termodinâmica Avançada 

## 🔥 Desafio 3

**Objetivo**: Realizar a combustão do composto escolhido.

-----


_Aspectos importantes a serem considerados:_

>Os potenciais termodinâmicos (energia interna, entalpia, energia livre de Gibbs) envolvidos na reação de combustão.

> A entropia envolvida na reação de combustão.

>A equação de estado a ser usada para o sistema.

------

In [3]:
# importações para o desafio
import numpy as np
from uncertainties import ufloat

<p style='text-align: justify'>Após os processos realizados no desafio 2 dos nossos experimentos em termodinâmica avançada, obtemos o 1-Hexeno em sua forma pura e líquida ao sair do condensador, o qual agora passará pela reação de combustão para poder entrar no ciclo rankine (no desafio 4). A reação de combustão corresponde à oxidação total de uma substância para geração de energia para um sistema, além disso essa reação gera também como produto água e gás carbônico. Ademais, é válido ressaltar que essa é reação é do tipo exotérmica, ou seja, a variação de entalpia da reação deve resultar em um número negativo, tendo, enfim, a liberação de energia.</p>

<p style='text-align: justify'>A reação de combustão em que iremos realizar no experimento, como dito anteriormente, é a do 1-Hexeno que pode ser vista abaixo já balanceada (considerando o 1-Hexeno no estado líquido):</p>

<p style='text-align: center'> $ C_{6}H_{12} + 9 \: O_{2} → 6 \: H_{2}O + 6\: CO_{2} $

#### Entalpia

<p style='text-align: justify'>Para darmos início nos cálculos do nosso experimento, vamos pensar em uma propriedade de suma importância para a medição da variação de energia na forma de calor. Pensando nas reações químicas, podemos denotar a mudança de entalpia associada a elas por $ Δ_{r}H $, que vai nos dizer se uma reação está liberando ou absorvendo energia (na forma de calor) durante seu processo e a quantidade respectiva.</p>
<p style='text-align: justify'>No nosso caso, temos uma reação de <b>combustão</b>, o que implica em ser uma reação <b>exotérmica</b>, ou seja, que libera calor (chamado de calor de combustão) na queima do combustível.</p>
<p style='text-align: justify'>Nos embasando em todo um estudo feito pelo livro "Physical Chemistry: A Molecular Approach", de McQuarrie [segunda referência], temos que:<p>

>A variação de entalpia para uma reação química pode ser vista como a entalpia total dos produtos menos a entalpia total dos reagentes.

<p style='text-align: center'>$Δ_{r}H = H_{produtos} - H_{reagentes}$

>Tendo em vista que, para reações exotérmicas (como as de combustão), em que a energia é liberada como calor, $H_{produtos} < H_{reagentes}$ , temos:

<p style='text-align: center'>$Δ_{r}H < 0$

>Sabendo, também, que a entalpia se caracteriza como uma função de estado $(H = U + PV ; ΔH = ΔU + PΔV)$, a qual se recorre quando temos a condição de pressão constante (condição frequente para as reações químicas que ocorrem de forma aberta à atmosfera), temos que a propriedade de aditividade se aplica à ela. Assim, se temos duas equações químicas e as somamos, resultando em uma equação global, como se fosse um processo dividido em duas etapas, podemos simplesmente somar a entalpia de ambos, já que: 

<p style='text-align: center'>$Δ_{r}H = Δ_{r}H(3) = Δ_{r}H(1) - Δ_{r}H(2)$

>Essa propriedade é conhecida como **Lei de Hess**. E, a partir dela, supondo, por exemplo, uma reação que de um composto obtemos dois elementos e outra inversa, que dois elementos se forma um composto, também conseguimos dizer que:

<p style='text-align: center'>$Δ_{r}H(reversa) = -Δ_{r}H$

<p style='text-align: justify'>Ainda sobre entalpia de reação, temos que, seu valor depende do número de mols dos reagentes. Nesse sentido, como há diversas possibilidades que poderiam ser ditas sobre a entalpia de uma reação, há alguns modos que padronizam a entalpia. Assim:</p>

>$Δ_{r}H^{0}$ : variação de entalpia associada a um mol de um reagente específico quanto todos os produtos e reagentes estão em seus estados padrão. Pensando em gas, equivaleria a um gas ideal na pressão de 1 bar em uma temperatura de interesse. (obs: medida intensiva, enquanto $Δ_{r}H$ é extensiva)

>$Δ_{f}H^{0}$ : variação de entalpia que padroniza a entalpia de reação de formação de um mol de uma mólecula a partir de seus elementos constituintes, estando todos os reagente e produtos nos seus estados padrão. (medição intensiva e extremamente útil)

<p style='text-align: justify'>Nesse sentido, uma variedade de subscritos podem compor a denotação de uma ampla gama de entalpia.</p>

<p style='text-align: justify'>De posse de toda essas informações, vamos calcular a entalpia da nossa reação de combustão de 1-Hexeno:</p>

<p style='text-align: center'> $ C_{6}H_{12}(l) + 9 \: O_{2}(g) → 6 \: H_{2}O(l) + 6\: CO_{2}(g) $

<p style='text-align: justify'>Tendo conhecimento da propriedade de aditividade da entalpia (lei de Hess) e da utilidade das energias de formação ($Δ_{f}H^{0}$), podemos pensar da seguinte forma:</p>

<ul>
    <li>Sendo uma equação química qualquer, em que a, b, y, e z são respectivos números de mols:
</ul>
    
<p style='text-align: center'> $ a \: A + b \: B → y \: Y + z \: Z $</p>

<ul>    
    <li>Podemos calcular $Δ_{r}H$ em duas etapas: (1) Decompomos A e B em seus elementos contituintes; (2) Combinamos os elementos para formar os compostos Y e Z. Assim:
</ul>

<p style='text-align: center'>$Δ_{r}H(1) = -aΔ_{f}H^{0}[A] - bΔ_{f}H^{0}[B]$</p>
    
<p style='text-align: center'>$Δ_{r}H(2) = yΔ_{f}H^{0}[Y] + zΔ_{f}H^{0}[Z]$</p>

<ul>    
    <li>Para a equação geral (lembrando que aqui é necessário que especifiquemos o estado físico de cada substância, por conta da entalpia de formação), temos então:
</ul>

<p style='text-align: center'>$Δ_{r}H =yΔ_{f}H^{0}[Y]+zΔ_{f}H^{0}[Z]-aΔ_{f}H^{0}[A]- bΔ_{f}H^{0}[B] $</p>

<ul>    
    <li>O que nos leva a:
</ul>

<p style='text-align: center'>$Δ_{r}H = H_{produtos} - H_{reagentes}$</p>

<p style='text-align: justify'>Aplicando para o nosso experimento:</p>

In [5]:
# definindo as energias de formação, coletadas do bando de dados NIST
delta_f_H_0_1_hexeno = ufloat(-73, 3) # kJ/mol
delta_f_H_0_O2 = 0 # pelo o O2 estar na sua forma padrão pura, sua energia de formação é 0
delta_f_H_0_H2O = ufloat(-285.83, 0.04) # kJ/mol
delta_f_H_0_CO2 = ufloat(-393.51, 0.13) # kJ/mol

In [7]:
# Para 1 mol de 1-Hexeno:
entalpia_produtos = 6 * delta_f_H_0_H2O + 6 * delta_f_H_0_CO2
entalpia_reagentes = 1 * delta_f_H_0_1_hexeno + 9 * delta_f_H_0_O2
entalpia_combustao_1_hexene = entalpia_produtos - entalpia_reagentes

print(f'A entalpia de combustão do 1-Hexeno (partindo da sua forma líquida) é {entalpia_combustao_1_hexene} kJ/mol')

A entalpia de combustão do 1-Hexeno (partindo da sua forma líquida) é -4003.0+/-3.1 kJ/mol


<p style='text-align: justify'><b>Com isso, para que a reação de combustão de 1-mol de 1-Hexeno possa ocorrer para se entrar no ciclo de Rankine, é necessário que seja fornecido, aproximadamente, 4003 kJ de energia.</b></p>

#### Entropia de combustão

<p style='text-align: justify'>Para dar sequência ao experimento, é necessário, enfim, calcular a entropia de combustão. Ao ter conhecimento do seu valor, poderemos compreender as propriedades termodinâmicas da reação, permitindo possíveis estudos para otimização dos processos relacionados a esse tipo de reação. Esse cálculo será realizado com valores de entropia encontradas em tabelas do site NIST, citado nas referências do experimento.</p>

In [10]:
S_hex = 295.2 #Entropia do 1-Hexeno líquido
S_O2 =  205 #Entropia do oxigênio
S_H2O = 6.6 #Entropia da água
S_CO2 = 120.4 #Entropia do gás carbônico

<p style='text-align: justify'>O cálculo pode, então, ser feito</p>

<p style='text-align: center'> $ a \: A + b \: B → c \: C + d \: D $</p>

<p style='text-align: center'> ΔS = (c*S[C] + d*S[D]) - (a*S[A] + b*S[B])</p>

<p style='text-align: justify'>ou seja, vamos fazer a soma das entropias dos produtos e a soma da entropia dos reagente e depois, a diferença existente entre os resultados.</p>

In [11]:
S_prod = S_H2O + S_CO2 # soma das entropias dos produtos
S_reag = S_hex + S_O2 # soma das entropias dos reagentes

S_combustao = S_prod - S_reag # variação da entropia

print(f'A variação da entropia de combustão para o 1-Hexeno será de {S_combustao} J/mol K')

A variação da entropia de combustão para o 1-Hexeno será de -373.2 J/mol K


#### Energia interna

<p style='text-align: justify'>Seguindo no raciocínio dos potenciais termodinâmicos envolvidos na combustão, temos a energia interna do sistema. Pensando em um sistema termodinâmico em equilíbrio, ele trabalha de forma que a energia interna (U) seja minimizada e que a entropia (S) seja maximixada. Tendo esses dois parâmetros, $U(S, V, N_{1}, ..., N_{n})$ e  $S(U, V, N_{1}, ..., N_{n})$, que dependem de uma ampla gama de variáveis, temos a definição completa do sistema do ponto de vista termodinâmico. Entretanto, eles são parâmetros difíceis de serem medidos e, praticamente, impossíveis de serem controlados.</p>

<p style='text-align: justify'>Sabendo disso, cientificamente, uma abordagem adotada é tornar parâmetros intensivos (os quais conseguimos medir e controlar razoavelmente bem, como temperatura e pressão), que surgem a partir de relações/derivadas dos parâmetros extensivos, como variáveis independentes para descrever o sistema. Isso porque, tendo as múltiplas derivadas (tangentes) dos mais variados pontos do problema (hipersuperfície, no caso), podemos descrevê-lo por inteiro também.</p>

<p style='text-align: justify'>Nesse contexto, utilizamos a Transformada de Lejandre para obtermos a forma de mensurar cada potencial sendo descrito para o nosso sistema. A partir dela, obtemos a forma de calcularmos a entalpia e a energia livre de Gibbs, por exemplo.</p>

#### Energia livre de Gibbs

<p style='text-align: justify'>A energia livre de gibbs é mais um dos importantes potenciais termodinâmicos que estão envolvidos no experimento, ela determina a direção de um processo espontâneo para sistemas com temperatura e pressão constantes.</p>

<p style='text-align: justify'>Para mensurar esse potencial, deve-se calcular ΔG = ΔH - TΔS, ou seja, a subtração da entalpia pela temperatura do sistema em kelvin multiplicada pela entropia do sistema, a qual foi obtida mais a cima no experimento. Essa equação foi obtida através da aplicação da transformada de Lejandre na equação da energia interna sistema considerando a entropia, pressão constante e volume constante.</p>

<p style='text-align: justify'>Todos os sistemas físicos tendem sempre a se mover na direção de equilíbrio, ou seja, de menor energia, em que a derivada da energia de Gibbs seja nula e, nesse caso, a entropia do sistema será máxima tendo uma grande multiplicidade. A depender do valor obtido ao se calcular esse potencial termodinâmico, é possível saber se a reação ocorre de maneira espontânea, sua variação é negativa, ou forçada, sua variação é positiva.</p>

<p style='text-align: justify'>Agora vamos realizar o cálculo para o 1-Hexeno, utilizando os valores obtidos a cima, para entender como o sistema se comporta durante a combustão,</p>

In [12]:
Gibbs_hex = (entalpia_combustao_1_hexene) - (298*S_combustao) 

print(f'A energia livre de Gibbs da combustão do 1-Hexeno é de {Gibbs_hex} J/mol')

A energia livre de Gibbs da combustão do 1-Hexeno é de 107210.6+/-3.1 J/mol


<p style='text-align: justify'>Com esse resultado positivo, percebe-se, então, o comportamento <b>não espontâneo da reação</b> e, sendo assim, o <b>1-Hexeno</b> pode ser tomado como <b>não inflamável</b>.<p>

#### Equação de estado do sistema

<p style='text-align: justify'>Uma equação de estado descreve o comportamento de um sistema termodinâmico a depender de suas variáveis com a temperatura, pressão e volume. Para resolvê-la, o melhor seria utilizar a equação de Van-der-Waals, que é própria para gases reais, pois considera as interações existentes entre as moléculas do sistema.</p>

<p style='text-align: center'> $$P + \frac{an^{2}}{V^{2}} * (V -nb) = nRT$$

<p style='text-align: justify'>Sendo P a pressão, V o volume molar, n o número de mols, R a constante dos gases ideais, T a temperatura em Kelvin e a e b constantes específicas da substância. No entanto, não será possível utilizá-la devido a falta de dados que permitam descobrir quais são as constantes a e b para o 1-Hexeno e, sendo assim, iremos utilizar a clássica equação dos gases ideais no nosso experimento.</p>

<p style='text-align: center'>$PV = nRT$

<p style='text-align: justify'>Nesse caso, passamos a considerar o comportamento da substância em condições ideais, dessa forma, ela não considera a interação entre as moléculas e considera que o volume de ocupação delas é insignificante em relação ao volume total do sistema. Essa aproximação é válida apenas quando as interações entre as moléculas são desprezíveis, mas dada a falta de informações para calcular a equação de estado de outra forma, como relatado a cima, iremos fazê-la dessa forma para descobrir o volume da substância para combustão,</p>

In [17]:
P = 100000 # pressão em Pa (considerando 1 bar)
n = 1 # número de mols
R = 8.31 # J/(mol*K)
T = 298 # temperatura em Kelvin

V = (n*R*T)/P

print(f'O volume de combustão é igual a {V} m^3')

O volume de combustão é igual a 0.024763800000000002 m^3


Sendo assim, a equação de estado do 1-Hexeno em condições ideais é:

<p style='text-align: center'> 100000 * 0.0247638 = 1 * 8.31 * 298</p>

### Conclusão

<p style='text-align: justify'>De posse de todas essas informações, podemos resumi-las para podermos passar ao próximo desafio preparados!</p>

<p style='text-align: justify'>Nosso objetivo aqui era, tendo obtido o 1-Hexeno puro e na sua forma líquida, realizar a combustão do mesmo, seguindo a equação:</p>

<p style='text-align: center'> $ C_{6}H_{12}(l) + 9 \: O_{2}(g) → 6 \: H_{2}O(l) + 6\: CO_{2}(g) $</p>

<p style='text-align: justify'>Para essa reação, então, calculamos os seguintes parâmetros que descrevem esse sistema:</p>

In [15]:
print(f'Entalpia de combustão do 1-Hexeno (partindo da sua forma líquida): {entalpia_combustao_1_hexene} kJ/mol')
print(f'Variação da entropia de combustão para o 1-Hexeno: {S_combustao} J/mol K')
print(f'Energia livre de Gibbs da combustão do 1-Hexeno: {Gibbs_hex} J/mol')
print(f'Volume de combustão: {V} m^3')

Entalpia de combustão do 1-Hexeno (partindo da sua forma líquida): -4003.0+/-3.1 kJ/mol
Variação da entropia de combustão para o 1-Hexeno: -373.2 J/mol K
Energia livre de Gibbs da combustão do 1-Hexeno: 107210.6+/-3.1 J/mol
Volume de combustão: 0.024763800000000002 m^3


<p style='text-align: justify'>Obtendo a seguinte equação de estado, considerando com comportamento ideal:</p>

<p style='text-align: center'> $100000 * 0.0247638 = 1 * 8.31 * 298$</p>

### Referências:

- SCHROEDER, Daniel V. An introduction to thermal physics. San Francisco, CA: Addison Wesley, 2000.

- MCQUARRIE, Donald A.; SIMON, John D. Physical chemistry: a molecular approach. Sausalito, Calif: University Science Books, 1997.

- 1-Hexene - NIST web chem book. Disponível em: <https://webbook.nist.gov/cgi/cbook.cgi?ID=C592416&Units=SI&Mask=7>